mne_bids/path.py

```diff
@@ -540,11 +603,26 @@ def make_bids_basename(subject=None, session=None, task=None,
     bids_path = BIDSPath(subject=subject, session=session, task=task,
                          acquisition=acquisition, run=run,
                          processing=processing, recording=recording,
-                         space=space, prefix=prefix, suffix=suffix)
+                         space=space, split=split, prefix=prefix,
+                         kind=kind, extension=extension)
     bids_path._check()
     return bids_path
 
 
+def _get_kind_ext_from_suffix(suffix):
+    """Parse suffix for valid kind and ext."""
+    # no matter what the suffix is, kind and extension are last
+    kind = suffix
+    ext = None
+    if suffix is not None:
```

Comment:
is it actually possible that it is None? 

apparently not. I tried to add:

`assert suffix is not None`

is no test broke. So I think it's safe to avoid this is `if`